Caltech-IPAC/firefly
# DM-12553: Allow automatic switching between FITS and HiPS

- Switch between image and FITS automatically
 - Show image outline drawing layer when switching to HiPS from image
 - Added support for HiPS cube
 - Refactor ZoomUtil code to be ZoomUtil and ZoomTask
 - Refactor ImagePlotCntlr to break out more reducer code into HandlePlotAdmin
 - Refactor PlotImageTask to break out PlotHipsTask


_To Test_:
 - FITS/HiPS switching: bring at catalogs up, zoom out and the image will turning into a HiPS
 - Cube test: bring a hiPS in that is a cube, you should see cube switch controls in the context toolbar.  Cube switching might be slow depend on the CDS server.
